src/kixi/datastore/schemastore/conformers.clj

```diff
@@ -141,10 +144,26 @@
     (string? x) (Boolean/valueOf (str x))
     :else :clojure.spec/invalid))
 
-(def bool? (s/conformer -bool?))
+(def bool? (s/with-gen (s/conformer -bool?)
+             (constantly (gen/boolean))))
 
 (defn -string?
   [x]
   (cond
     (string? x) x
     :else (str x)))
+
+(defn timestamp?
+  [s]
+  (tf/parse
+   (tf/formatters :basic-date-time)
+   s))
+
+(def uuid?
+  (partial re-find #"^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"))
+
+(def uuid (s/with-gen uuid?
+            #(gen/fmap str (gen/uuid))))
+
+(def anything (s/with-gen (constantly true)
```

Comment:
Well it's a bit hacky, but the (constantly true) has the effect of saying everything is valid. So the value can be anything! In terms of conforming it won't do anything.